### ISSUE TITLE
Doesn't install on VS2015

### ISSUE THREAD

**Issue by Threetwosevensixseven** [chingucoding/IndentRainbow#3]
![image](https://user-images.githubusercontent.com/12496735/55415667-6d7f4700-553b-11e9-9006-10d17c85f35a.png)

**Comment by Threetwosevensixseven** [comment]
Here is the install log:

[VSIXInstaller_d8a0bc9e-6078-48aa-949f-afa8d864171a.log](https://github.com/chingucoding/IndentRainbow/files/3035246/VSIXInstaller_d8a0bc9e-6078-48aa-949f-afa8d864171a.log)

**Comment by chingucoding** [comment]
Have you tried installing version 0.2.2 of the extension?
With version 0.2.2 , MPF 15.0 was removed as dependency and it should work in Visual Studio 2015.

**Comment by Threetwosevensixseven** [comment]
Thank you! I see what happened. The Extension Markplace download link wasn't working earlier, so I came to github and installed the latest release, which is 0.2.

Now it is installs without an error, but I cannot set options for the extension.
![image](https://user-images.githubusercontent.com/12496735/55424686-d328fe80-554e-11e9-8657-ec6662227e29.png)

I uninstalled and reinstalled twice, and got the same problem every time.
